src/verilog/verilog_typecheck.cpp

```diff
@@ -1016,51 +1016,46 @@ void verilog_typecheckt::convert_assert_assume_cover(
   convert_expr(cond);
   make_boolean(cond);
 
-  if(
-    module_item.id() == ID_verilog_assert_property ||
-    module_item.id() == ID_verilog_cover_property)
-  {
-    // We create a symbol for the property.
-    // The 'value' of the symbol is set by synthesis.
-    const irep_idt &identifier = module_item.identifier();
+  // We create a symbol for the property.
+  // The 'value' of the symbol is set by synthesis.
+  const irep_idt &identifier = module_item.identifier();
 
-    irep_idt base_name;
+  irep_idt base_name;
 
-    if(identifier == irep_idt())
-    {
-      assertion_counter++;
-      base_name = std::to_string(assertion_counter);
-    }
-    else
-      base_name = identifier;
+  if(identifier == irep_idt())
+  {
+    assertion_counter++;
+    base_name = std::to_string(assertion_counter);
+  }
+  else
+    base_name = identifier;
 
-    std::string full_identifier =
-      id2string(module_identifier) + ".property." + id2string(base_name);
+  std::string full_identifier =
+    id2string(module_identifier) + ".property." + id2string(base_name);
 
-    if(symbol_table.symbols.find(full_identifier) != symbol_table.symbols.end())
-    {
-      throw errort().with_location(module_item.source_location())
-        << "property identifier `" << base_name << "' already used";
-    }
+  if(symbol_table.symbols.find(full_identifier) != symbol_table.symbols.end())
+  {
+    throw errort().with_location(module_item.source_location())
+      << "property identifier `" << base_name << "' already used";
+  }
 
-    module_item.identifier(full_identifier);
+  module_item.identifier(full_identifier);
 
-    symbolt symbol{base_name, bool_typet{}, mode};
+  symbolt symbol{base_name, bool_typet{}, mode};
 
-    symbol.module = module_identifier;
-    symbol.value = nil_exprt{}; // set by synthesis
-    symbol.name = full_identifier;
-    symbol.is_property = true;
-    symbol.location = module_item.find_source_location();
-    symbol.pretty_name = strip_verilog_prefix(full_identifier);
+  symbol.module = module_identifier;
+  symbol.value = nil_exprt{}; // set by synthesis
+  symbol.name = full_identifier;
```

Comment:
It needs to be `symbolt symbol{full_identifier, bool_typet{}, mode};` and then `symbol.base_name = base_name;`